app/locker/repository/Statement/EloquentRepository.php

```diff
@@ -28,7 +28,12 @@ public function __construct() {
    * @return [String] UUIDs of the stored statements.
    */
   public function store(array $statements, array $attachments, array $opts) {
-    return $this->storer->store($statements, $attachments, new StoreOptions($opts));
+    $result = $this->storer->store($statements, $attachments, new StoreOptions($opts));
+
+    // The statement.store event is used the message queue system
+    $event = \Event::fire('statement.store', array($statements));
```

Comment:
I don't think statements here are actually the final version of the statements that we store.